scripts/scenes.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS scenes;
+
+CREATE TABLE scenes (
+  id TEXT NOT NULL PRIMARY KEY, 
+  value JSONB,
+  email TEXT NOT NULL,
+  title TEXT NOT NULL,
+  description TEXT,
+  address TEXT,
+  parcels_count INTEGER,
+  triangles_count INTEGER,
+  items_count INTEGER,
+  sessions INTEGER,
+  days INTEGER,
+  n_events INTEGER,
+  create_time INTEGER,
+  tag TEXT,
+  created_at TIMESTAMP,
+  updated_at TIMESTAMP
```

Comment:
I think this looks different from the type https://github.com/decentraland/builder-contest-server/pull/14/files#diff-60e1ce117a51d7af5dd8d7072f3d7070R10